### DISCUSSION TITLE
What report contains detailed FBA refund information?

### DISCUSSION THREAD

**Discussion by benzitney** [amzn/selling-partner-api-models#4300] [Q&A] [answered]
I'm looking to get refund information down to the order-item level, like in `GET_XML_RETURNS_DATA_BY_RETURN_DATE`, but for FBA orders. (My understanding is GET_XML_RETURNS_DATA_BY_RETURN_DATE only contains FBM, but please correct me if I'm wrong.)

I'm looking at `GET_FBA_FULFILLMENT_CUSTOMER_RETURNS_DATA`, but this doesn't have refund information.

**Comment by ritikacd** [comment, marked as answer]
Hi @benzitney 

You can use [Settlement Reports](https://developer-docs.amazon.com/sp-api/docs/report-type-values-settlement) or [Finance APIs](https://developer-docs.amazon.com/sp-api/docs/finances-api-reference) to get refunds information per order level. 

Please note that the Settlement reports cannot be requested or scheduled. They are automatically scheduled by Amazon (usually every 14 days) and are tied to the disbursement cycle of a seller account.

You can use getReports API method with the ReportType parameter set to “Settlement” to check when the last settlement report was available. Additionally, you can use notifications to be notified when a settlement report is available. This can be set up on a 14-day schedule to coincide with the availability of the reports. 

[listFinancialEvents](https://developer-docs.amazon.com/sp-api/docs/finances-api-reference#listfinancialevents) operation under Finance API returns different transaction events applicable to seller's transaction, provide delta date query to filter on dates and use `RefundEventList` event to see the refund transactions. 


Thanks,
Ritika
Selling Partner API Team
